Kwartracker/Scenes/My Profile Page/Content/MyProfileContentView.swift

```diff
@@ -36,13 +41,16 @@ struct MyProfileContentView_Previews: PreviewProvider {
 }
 
 private struct MyProfileCardView: View {
-    let name: String = L10n.ProfilePage.dummyName
+    let user: FetchProfileQuery.Data.Profile?
+
     private let cardHeight: CGFloat = 190
     private let fontSize: CGFloat = 15
     private let horizontalMargin: CGFloat = 30
     private let verticalMargin: CGFloat = 10
     
     var body: some View {
+        let name = "\(self.user?.firstName ?? "") \(user?.middleName ?? "") \(user?.lastName ?? "")"
+        
```

Comment:
```suggestion
    private var name: String {
        "\(self.user?.firstName ?? "") \(user?.middleName ?? "") \(user?.lastName ?? "")"
    }
    
    var body: some View {
    }
```
It's better to create variables outside body or any view variable